test/bin/scenario.sh

```diff
@@ -801,11 +802,12 @@ run_tests() {
     record_junit "${vmname}" "robot_framework_installed" "OK"
 
     # Make sure oc command is available
-    if ! command -v oc &> /dev/null
-    then
-        error "OpenShift Client package not installed, install it with ${ROOTDIR}/scripts/fetch_tools.sh oc"
-        record_junit "${vmname}" "oc_installed" "FAILED"
-        exit 1
+    if ! command -v oc &> /dev/null ; then
+        echo "OpenShift Client package not installed, installing with ${ROOTDIR}/scripts/fetch_tools.sh oc"
```

Comment:
Let's remove the echo to be consistent with other places?

Reply:
Done